### ISSUE TITLE
Code Reviews

### ISSUE THREAD

**Issue by ngolapnguyen** [dwarvesf/ts-tokyoclient#11]

**Comment by haongo138** [comment]
> Not sure why gamepad is kept here? I think it should be in the tokyo-gamepad project

Currently, in lib, user has two choices: 
- Implement an algorithm in `setGamePlan` to consume & respond to event from the map (sent from server) by using `gamepad` obj
- Manually controls the ship via `tokyo-gamepad` using exposed `gamepad` object

**Comment by haongo138** [comment]
> Why is EventType an enum? Shouldn't it be a type/interface?

I think `EventType` is only used as a constant represents type of event from server

**Comment by haongo138** [comment]
> Type/interface names should be more consistent. E.g. IConfig -> Config

yep, I will update this one after done discussing about two questions above
